.travis.yml

```diff
@@ -4,8 +4,19 @@ scala:
   - 2.11.8
   - 2.12.1
 
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot
```

Comment:
Cool, I got the commands from https://github.com/typesafehub/zinc/blob/master/.travis.yml. Happy to know we also cache launcher the way you propose.